### PR TITLE
Add Indicator Fields

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
In Hold(Won't pass until `demisto-sdk` file validation is fixed)

## Related Issues
Efforts toward this issue: https://github.com/demisto/etc/issues/19709

## Description
Adds 'associations', 'port', 'region', 'reportedby', 'reputation', 'status', 'threattypes' and 'trafficlightprotocol' indicator fields.

## Required version of Demisto
5.5.0

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: IndicatorFields/incidentfield-port.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "mergeStrategy": "",
+    "hidden": false,
+    "associatedTypes": [
+        "IP",
+        "IPv6"
```
Comment (DeanArbel) [inline IndicatorFields/incidentfield-port.json]: This indicator type doesn't exist yet. The PR is dependent on it.

Comment (avidan-H) [reply IndicatorFields/incidentfield-port.json]: and on CIDR and CIDRv6

**Comment by avidan-H** [comment]
The `First Seen` indicator field also still needs to be sorted out.

**Comment by avidan-H** [comment]
Spoke to @Nitzperetz, and the `First Seen` indicator field is a pre-existing built-in field, so that is not a new field that needs to be defined on the content side. Furthermore, the `Reputation` indicator field is being implemented server-side and can be dropped from here.

**Comment by avidan-H** [comment]
@kirbles19 Going to skip the technical writer's review on the indicator fields' descriptions for now so we can merge and get on with the issues waiting on this PR with the understanding that the descriptions will still undergo a review and get updated accordingly if necessary.

**Comment by avidan-H** [comment]
@kirbles19 please take a look at the indicator descriptions when you get a chance though and we can open a new PR for any edits you may have.

**Comment by kirbles19** [comment]
Removing docs-approved label, I haven't reviewed this yet.

File: IndicatorFields/incidentfield-reportedby.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "sortValues": null,
+    "propagationLabels": null,
+    "vcShouldIgnore": false,
+    "commitMessage": "",
+    "shouldCommit": false,
+    "name": "Reported By",
+    "prevName": "Reported By",
+    "ownerOnly": false,
+    "placeholder": "",
+    "description": "Source which reported this indicator. Can include Feeds, Incidents, Users",
```
Comment (kirbles19) [inline IndicatorFields/incidentfield-reportedby.json]: The source that reported this indicator. Cane be a feed, an incident, or users.

**Comment by kirbles19** [comment]
@avidan-H Done. 1-2 comments that need to be implemented.

**Pull request merged**